ITR/temperature_score.py

```diff
@@ -53,6 +73,11 @@ def get_score(self, scorable_row: pd.Series) -> Tuple[
                         TARGET_SCORE, TARGET_OVERSHOOT, TEMPERATURE_RESULTS])
         """
 
+        weighted_budget = self._get_weighted_budget(q_s1s2=scorable_row[self.c.COLS.GHG_SCOPE12],
+                                                    q_s3=scorable_row[self.c.COLS.GHG_SCOPE3],
+                                                    scope=scorable_row[self.c.COLS.SCOPE],
+                                                    global_budget=scorable_row[self.c.COLS.BENCHMARK_GLOBAL_BUDGET])
+
```

Comment:
I apologize that the code is confusing, due to the badly named (and probably badly conceived) instance variables.  When setting up the fundamental company data, it was decided that the ICompanyData would contain BASE_YEAR_PRODUCTION, GHG_SCOPE12, and GHG_SCOPE3, which are all point-in-time measurements of production and emissions.  But those points in time are NOT cumulative budget, which is what the temperature score and weighting factors are based upon.  The ONLY real purpose I can think of for these terms is in the initial setup of the production and EI projections.  For this reason, there's really no reason to leave them lying around in the ICompanyData object.  Instead, they should be variables conveyed to initialize projections, and then promptly forgotten.  If you want to know base year statistics, query the data on the continuum of historic to future projections.  If you want to know cumulative budget--that's what the Data Warehouse computes and retains.

Reply:
from your comment, i cannot understand, what you suggest me to do

Reply:
This also related to cumulative emissions needed for weighting vs. initial value needed for projections.

Reply:
Yes, this comment thread is an instance of our weights discussion.
Since we track this discussion in a separate comment thread, I will close this duplicating thread as resolved